shortstories/forms.py

```diff
@@ -1,11 +1,10 @@
 from  django import forms
 from .models import ShortStory
-from  django.core import validators
 
 class ShortStoryForm (forms.ModelForm):
 
-    body = forms.CharField(widget =forms.Textarea(attrs={'maxlength':'35000',}))
+    body = forms.CharField(widget=forms.Textarea(attrs={'maxlength':'35000',}))
 
     class Meta:
           model = ShortStory
-          fields = ('author','title','status','body','publication_date','minutes_to_read')
+          fields = ('author', 'title', 'status', 'body', 'publication_date', 'minutes_to_read')
```

Comment:
why are you including `minutes_to_read`? I think even we can drop the `publication_date`.

Reply:
I got this error when I removed publication_date "null value in column "publication_date" violates not-null constraint", so I added again.